From cb887ab2b73f60c386cca6f185e789b9c252cb5e Mon Sep 17 00:00:00 2001
From: Qiu Feng <feng.qiu@intel.com>
Date: Thu, 9 Jul 2020 23:44:00 +0800
Subject: [PATCH] Fix the build error in "make psw_install_pkg"

Signed-off-by: Qiu Feng <feng.qiu@intel.com>
---
 Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Makefile b/Makefile
index 5fb368a9..35253364 100644
--- a/Makefile
+++ b/Makefile
@@ -96,6 +96,10 @@ sdk_install_pkg: sdk
 	./linux/installer/bin/build-installpkg.sh sdk cve-2020-0551
 
 psw_install_pkg: psw
+ifeq ("$(wildcard ./external/dcap_source/QuoteGeneration/psw/ae/data/prebuilt/libsgx_qe3.signed.so)", "")
+	./external/dcap_source/QuoteGeneration/download_prebuilt.sh
+	$(CP) external/dcap_source/QuoteGeneration/psw/ae/data/prebuilt/libsgx_qe3.signed.so $(BUILD_DIR)
+endif
 	./linux/installer/bin/build-installpkg.sh psw
 
 .PHONY: deb_libsgx_ae_qe3
-- 
2.24.3 (Apple Git-128)

