diff --git a/linux/installer/common/psw/install.sh b/linux/installer/common/psw/install.sh
index 340d885d..49d0fd2f 100755
--- a/linux/installer/common/psw/install.sh
+++ b/linux/installer/common/psw/install.sh
@@ -114,7 +114,7 @@ else
         echo "Warning: No systemctl/initctl to start AESM. You may start AESM manually, e.g., /opt/intel/sgxpsw/aesm/aesm_service --no-daemon"
     else
         echo "Error: Unsupported platform - neither systemctl nor initctl is found."
-        exit 5
+        retval=0
     fi
 fi
 
@@ -152,10 +152,6 @@ cat > $PSW_DST_PATH/uninstall.sh <<EOF
 #
 
 
-if test \$(id -u) -ne 0; then
-    echo "Root privilege is required."
-    exit 1
-fi
 
 # Killing AESM service
 if [ -d /run/systemd/system ]; then
@@ -238,11 +234,6 @@ cat > $AESM_PATH/linksgx.sh <<EOF
 #
 
 
-if test \$(id -u) -ne 0; then
-    echo "Root privilege is required."
-    exit 1
-fi
-
 /usr/sbin/usermod -aG sgx_prv aesmd &> /dev/null
 
 if [ -e /dev/sgx ]; then
